### ISSUE TITLE
503 on data load for Amazon items

### ISSUE THREAD

**Issue by 004a** [Wingysam/Christmas-Community#2]
```
Version Info:
Christmas Community: v1.27.0
Get Product Data: v1.27.0
Node: v15.14.0
PID: 72
```

I am receiving the following 503 error when attempting to add new items or refresh items from Amazon.

```text
/usr/src/app/node_modules/get-product-name/sites/amazon.js:90
    if (!res.ok) throw new Error(`Res not ok. Status: ${res.status} ${res.statusText}`)
                       ^

Error: Res not ok. Status: 503 Service Unavailable
    at getter (/usr/src/app/node_modules/get-product-name/sites/amazon.js:90:24)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async fetchData (/usr/src/app/node_modules/get-product-name/index.js:6:16)
    at async /usr/src/app/routes/wishlist/index.js:260:27
```

**Comment by Wingysam** [comment]
Sorry I didn't get to it in time and now the link is dead, could you reupload it? thanks :)

**Comment by vityav** [comment]
Hello, I'm running into the same issue. Amazon links autofill everything but the price, a plain curl returns a notice about automated data access, and a curl with the specified useragent returns a never ending binary stream. I tried setting `GPD_AMAZON_HTML` in the `.env` file but didn't get any logs out when adding an amazon link. Does that variable work for the latest (today, 11/28/21) docker image? Are there any other logs I can check? All etsy links fail to gather names or prices, so it may be an issue on my end.

**Comment by Wingysam** [comment]
What are you setting the env variable to? It should be a path to save the html to.

**Comment by vityav** [comment]
I figured out the issue was that docker compose wasn't loading my `.env` file, and manually defining the `GPD_AMAZON_HTML` env variable in a shell in the container wasn't being read for some reason. Attached is the output. 
[amazon.txt](https://github.com/Wingysam/Christmas-Community/files/7622362/amazon.txt)

**Comment by Wingysam** [comment]
Thank you! I should be able to get this working tomorrow.

**Comment by vityav** [comment]
Thanks! And for reference/searchability, that output was created adding this link:
https://smile.amazon.com/dp/B07GBXMBQF

**Comment by Wingysam** [comment]
Sorry I wasn't able to fix it on the 1st, looking into this now.

**Comment by Wingysam** [comment]
This has been fixed in get-product-name [#b60f9924](https://gitlab.com/wingysam/get-product-name/-/commit/b60f99243bcfa8107fc845a538d59313150ef6cb).
